relays/bin-substrate/src/messages_lane.rs

```diff
@@ -201,6 +201,15 @@ pub struct StandaloneMessagesMetrics {
 	pub source_to_base_conversion_rate: Option<F64SharedRef>,
 }
 
+impl StandaloneMessagesMetrics {
+	/// Return conversion rate from target to source tokens.
+	pub async fn target_to_source_conversion_rate(&self) -> Option<f64> {
+		let target_to_base_conversion_rate = (*self.target_to_base_conversion_rate.as_ref()?.read().await)?;
+		let source_to_base_conversion_rate = (*self.source_to_base_conversion_rate.as_ref()?.read().await)?;
+		Some(target_to_base_conversion_rate / source_to_base_conversion_rate)
```

Comment:
Might panic in case `source_to_base_conversion_rate` is `0`. Should we return `None` instead?

Reply:
Yes, thanks. It shouldn't panic, but any negative, zero or non-normal float values may cause some issues elsewhere. So I've added check to the metric itself, which reads these values.